src/Stack/Types/BuildPlan.hs

```diff
@@ -102,15 +102,15 @@ instance Store SnapshotDef
 instance NFData SnapshotDef
 
 snapshotDefVC :: VersionConfig SnapshotDef
-snapshotDefVC = storeVersionConfig "sd-v1" "tnwWSSLerZ2XeR6XpVwj5Uh0eF4="
+snapshotDefVC = storeVersionConfig "sd-v1" "CKo7nln8EXkw07Gq-4ATxszNZiE="
```

Comment:
I've no idea what this is - I was just following the compiler messages. Is this right?

Reply:
This does a hash of the type definitions involved in the snapshot cache.  `sd-v1` gets prefixed to the file as a version tag to ensure that the serialization matches up.  In this case it is fine to change the hash, because it is just a rename, the serialization has not changed.

We should probably have a comment on these documenting how they work.  Maybe referencing an md file.

Reply:
I see, thanks! Yes, seems like one for documenting :)